virttest/vt_resmgr/resources/storage/file_volume.py

```
@@ -0,0 +1,231 @@
+            }
+        )
+
+    def release(self, arguments, node):
+        node, backing_id = self.get_binding(node)
+        r, o = node.proxy.resource.update_resource_by_backing(
+            backing_id,
+            "release",
+            arguments,
+        )
+        if r != 0:
+            raise Exception(o["out"])
```

Ruff (0.13.1)

Create your own exception

(TRY002)